docs/integrations/builtin/credentials/mist.md

```diff
@@ -5,14 +5,27 @@ description: Documentation for the Mist credentials. Use these credentials to au
 
 # Mist credentials
 
-You can use these credentials to authenticate when using the HTTP Request node to make a [Custom API call](/integrations/custom-operations/).
+You can use these credentials to authenticate when using the [HTTP Request node](/integrations/builtin/core-nodes/n8n-nodes-base.httprequest/) to make a [Custom API call](/integrations/custom-operations/).
 
-## Authentication methods
+## Prerequisites
 
-n8n supports Mist authentication using an API token.
+Create a [Mist](https://www.mist.com/){:target=_blank .external-link} account and organization. Refer to [Create a Mist account and Organization](https://www.mist.com/documentation/create-mist-org/){:target=_blank .external-link} for detailed instructions.
+
+## Supported authentication methods
+
+- API token
 
 ## Related resources
 
-Refer to [Mist's documentation](https://www.mist.com/documentation/mist-api-introduction/){:target=_blank .external-link} for more information about the service.
+Refer to [Mist's documentation](https://www.mist.com/documentation/mist-api-introduction/){:target=_blank .external-link} for more information about the service. If you're logged in to your MIst account, go to [https://api.mist.com/api/v1/docs/Home](https://api.mist.com/api/v1/docs/Home){:target=_blank .external-link} to view the full API documentation.
 
 This is a credential-only node. Refer to [Custom API operations](/integrations/custom-operations/) to learn more. View [example workflows and related content](https://n8n.io/integrations/mist/){:target=_blank .external-link} on n8n's website.
+
+## Using API token
+
+To configure this credential, you'll need:
+
+- An **API Token**: You can use either a User API token or an Org API token. Refer to [How to generate a user API token](https://www.mist.com/documentation/using-postman/){:target=_blank .external-link} for instructions on generating a User API token. Refer to [Org API token](https://www.mist.com/documentation/org-api-token/){:target=_blank .external-link} for instructions on generating an Org API token.
+- Select the **Region** you're in. Options include:
+    - `Europe`: Select this option if your cloud environment is in any of the EMEA regions.
```

Comment:
Is "Europe" UI text? I'm wondering if this (and a few similar instances) should be bold rather than code formatted?